config.json

```diff
@@ -1,10 +1,11 @@
 // Hjson format, see http://hjson.org for syntax
 {
-    // The URL of the kernel gateway / jupyter notebook server to contact for
-    // kernels
+    // URL of the kernel gateway / jupyter notebook server to contact for kernels
     KERNEL_GATEWAY_URL: http://192.168.99.100:8888
     // Authorization token required for kernel gateway requests
     KG_AUTH_TOKEN: null
+    // Kernel gateway base URL (gets appended to KERNEL_GATEWAY_URL)
+    KG_BASE_URL: ""
```

Comment:
If it's just getting appended, we probably could have just said "point kernel gateway URL **directly** to the parent path where the /api resource lives". But, OK, live and learn.